scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
Why is this script added in a PR bumping ic-cdk? How did the integration tests work before?